offchain-modules/cli/src/commands/dapp/mod.rs

```diff
@@ -1,6 +1,8 @@
 use anyhow::Result;
 use types::*;
 
+use force_eth_lib::dapp::relayer::ckb_relayer::CkbTxRelay;
```

Comment:
add `pub use relayer::ckb_relayer::CkbTxRelay;` in dapp/mod.rs,  then you can use `use force_eth_lib::dapp:: CkbTxRelay` here